lightkurve/targetpixelfile.py

```diff
@@ -922,6 +923,14 @@ def get_model(self, star_priors=None, **kwargs):
         if 'prfmodel' not in kwargs:
             kwargs['prfmodel'] = self.get_prf_model()
         if 'background_prior' not in kwargs:
+            if np.all(np.isnan(self.flux_bkg)):  # If TargetPixelFile has no background flux data
+                # 5 sigma clip the TargetPixelFile flux until convergence
+                clipped_flux = sigma_clip(self.flux, sigma=5, iters=None)
+                bkg = []
+                for cadence in self.flux:
+                    # Set background flux as the median of the clipped flux for each cadence
+                    cadence_bkg = bkg.append(np.full(self.shape[1:],np.ma.median(cadence))
+                self.flux_bkg = np.stack(bkg, axis=0)
```

Comment:
Ohhh, you are deriving a background estimate for each cadence?  Does this work?  The PSF photometry tool right now assumes this prior is a single value, i.e. a constant background estimate.  For simplicity, `TPFModel` has been developed to model a single cadence.  What we could do is have `get_model` take a `cadence` parameter that estimates the background for that specific cadence.